ivy-data/src/main/java/com/ivy/data/model/TagAssociation.kt

```diff
@@ -0,0 +1,14 @@
+package com.ivy.data.model
+
+import com.ivy.data.model.primitive.AssociationId
+import com.ivy.data.model.primitive.TagId
+import com.ivy.data.model.sync.Syncable
+import java.time.Instant
+
+@Suppress("DataClassTypedIDs")
```

Comment:
Seems like you've fixed this error and can remove the suppress. If it doesn't work then it's on me and I gotta fix my Detekt rules 😅